components/brave_shields/browser/brave_shields_p3a.cc

```diff
@@ -152,16 +161,28 @@ void RecordShieldsDomainSettingCounts(PrefService* profile_prefs,
   const char* below_hg_name = is_fingerprint
                                   ? kDomainFPSettingsBelowHistogramName
                                   : kDomainAdsSettingsBelowHistogramName;
+  // Retrieve a count of domains with a setting above the global setting.
```

Comment:
It's not clear what "above" means. I think in this context you mean more or less permissive?

Reply:
Clarified on lines 105-106. The P3A wiki also describes "above"/"below": https://github.com/brave/brave-browser/wiki/P3A#q51-on-how-many-domains-has-the-user-set-the-adblock-setting-to-be-lower-block-less-than-the-default